glossary.yml

```diff
@@ -3987,25 +4060,19 @@
     term: "procedural programming"
     def: >
       A style of programming in which functions operate on data that is passed into
-      them. The term is used in contrast to [object-oriented programming](#oop).
+      them. The term is used in contrast to other programming styles, such as
+      [object-oriented programming](#oop) and [functional programming](#functional_programming).
 
 
 - slug: process
   en:
     term: "process"
     def: >
       An [operating system](#operating_system)'s representation of a running program.
-      A process typically has some memory, the identify of the user who is running it,
+      A process typically has some memory, the identity of the user who is running it,
       and a set of connections to open files.
 
 
-- slug: product_manager
```

Comment:
I would say it's better not to delete definitions, even if they are poorly defined because it would break backwards compatibility. Instead, add [FIXME] to the definition so that it's easier to identify as problematic.